guides/doc-Planning_for_Project/topics/Deployment_Considerations.adoc

```diff
@@ -190,7 +190,7 @@ Note that kickstart repositories should not be added to content views, as they a
 
 Content deployment is the management of errata and packages on content hosts.
 There are two methods for content deployment on {Project}.
```

Comment:
The section says REX is the default and the preferred method, but doesn't list any alternatives.